readthedocs/projects/notifications.py

```diff
@@ -11,3 +15,12 @@ class ResourceUsageNotification(Notification):
     context_object_name = 'project'
     subject = 'Builds for {{ project.name }} are using too many resources'
     level = REQUIREMENT
+
+
+class EmailConfirmNotification(SiteNotification):
+
+    failure_level = ERROR_PERSISTENT
+    failure_message = _(
+        'Add/Verify your primary email address. '
```

Comment:
I'd write this like:

    Please <a href="">add or verify your primary email address</a>.